mongodb-queue.js

```diff
@@ -266,3 +273,12 @@ Queue.prototype.done = function(callback) {
         callback(null, count)
     })
 }
+
+Queue.prototype._optionsWithNewDocument = function(query) {
```

Comment:
Maybe renaming the function to `getDefaultQueryOptions` might improve readability

Reply:
It's not _quite_ what this function does: it actually adds the "return a new document" flag (whatever that is) to the passed set of options.

If we had access to the spread operator you could do something like:

```js
var options = {
  // other options here
  ...self._returnOriginal(),
}
```

...but we can't really do that without a transpiler I think.

Reply:
Could just call it `_returnOriginal()` anyway. Probably just as meaningful with fewer words:

```js
var options = self._returnOriginal({
  // other options
})
```

Reply:
So I got a question, sorry for my ignorence, but why don't you just replace the flag and discard the previous/deprecated option? 

If the usage is deprecated, just replace it ?

Reply:
To avoid making this a breaking change — it would break any clients who are using older versions of `mongodb`.